v2/api/authorization/v1api20220401/storage/structure.txt

```diff
@@ -4,12 +4,15 @@ github.com/Azure/azure-service-operator/v2/api/authorization/v1api20220401/stora
 APIVersion: Enum (1 value)
 └── "2022-04-01"
 RoleAssignment: Resource
-├── Spec: Object (12 properties)
+├── Spec: Object (13 properties)
 │   ├── AzureName: string
 │   ├── Condition: *string
 │   ├── ConditionVersion: *string
 │   ├── DelegatedManagedIdentityResourceId: *string
 │   ├── Description: *string
+│   ├── OperatorSpec: *Object (2 properties)
+│   │   ├── NamingConvention: *string
```

Comment:
Is there anything actually reading/doing anything with this right now?

We shouldn't add this unless there's something honoring it. I think to do that we need to change the following places too:
v2/api/authorization/v1api20200801preview/role_assignment_defaults.go
v2/api/authorization/v1api20220401/role_assignment_defaults.go

Alternatively, we can back this change out for now and then include it in a separate PR focused on this (and maybe SQL role assignment, which has the same naming behavior).

Reply:
Agreed. This was a demo; I'll back it out of this PR.